### PR TITLE
Backend Copy Portfolio Routes

### PR DESCRIPTION
**Pull request by capture120**
# Description

Adds routes for copying a portfolio.
Adds dummy data for portfolios and positions.
Adds separate GetPortfolio Route for convenience.

# How Has This Been Tested?

copy route updates data in db

# Checklist

- [ ] I have performed a self-review of my code
- [ ] I have reached out to another developer to review my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: backend/src/services/portfolio.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+func NewPortfolioService(db *gorm.DB) *PortfolioService {
+	return &PortfolioService{
+		DB: db,
+	}
+}
+
+// CopyPortfolio copies the positions from the target portfolio into the current user's portfolio.
+func (os *PortfolioService) CopyPortfolio(currentUserPortfolio models.UserPortfolio, targetPortfolio models.UserPortfolio) (models.UserPortfolio, error) {
```
Comment (aniamisiorek) [inline backend/src/services/portfolio.go:20]: Are we copying the portfolio to the e-trade account?

File: backend/src/services/etrade.go [hunk 2]
```diff
@@ -18,8 +17,10 @@ const APIEnv = "apisb" // "apisb" = SANDBOX, "api" = PROD
 
 var oauthClient = oauth.Client{
 	Credentials: oauth.Credentials{
-		Token:  os.Getenv("OAUTH_KEY"),
-		Secret: os.Getenv("OAUTH_SECRET"),
+		// Token:  os.Getenv("OAUTH_KEY"),
```
Comment (aniamisiorek) [inline backend/src/services/etrade.go:20]: What's the reasoning for removing this?

Comment (capture120) [reply backend/src/services/etrade.go:20]: forgot to change that, will fix later tn

File: backend/src/services/portfolio.go (hunk 1 above)
Comment (capture120) [reply backend/src/services/portfolio.go:20]: this just has routes to copy portfolios in our own db for now. ETrade endpoints are not working so I'm looking into fixing that + how to make trade orders for securities with the api

**Review by leoRysing — APPROVED** [review]

**Pull request merged**